components/service/glean/build.gradle

```diff
@@ -14,7 +14,7 @@ apply plugin: 'kotlin-android'
  * created during unit testing.
  * This uses a specific version of the schema identified by a git commit hash.
  */
-String GLEAN_PING_SCHEMA_GIT_HASH = "6041aa6"
+String GLEAN_PING_SCHEMA_GIT_HASH = "6041aa6" // TO UPDATE BEFORE MERGE
```

Comment:
This schema hash will need to be updated once mozilla-services/mozilla-pipeline-schemas#245 is merged.